packages/react-storage/src/components/StorageManager/utils/filterAllowedFiles.ts

```diff
@@ -1,7 +1,16 @@
 export const filterAllowedFiles = (
   files: File[],
-  acceptedFileTypes: string[]
+  acceptedFileTypes?: string[]
 ): File[] => {
+  // Allow any files if acceptedFileTypes is undefined, empty array, or contains '*'
+  if (
+    !acceptedFileTypes ||
+    acceptedFileTypes.length === 0 ||
+    acceptedFileTypes.includes('*')
```

Comment:
Let's include an integration test for this.

Reply:
added a drag and drop integration test